web/src/styles/base/global.scss

```diff
@@ -209,6 +209,10 @@ ol {
   }
 }
 
+.usa-file-input {
+  max-width: unset;
+}
```

Comment:
Is this overriding the default? If so, did you check for sure it works / is actually overridden in browser? usually in order to to override an njwds setting, we have to use `!important`. I'm surprised to see this would work like this

Reply:
Yup, this is overriding the default value. This is pulling styles from the USWDS package. I'm not sure how NJWDS implements USWDS.
![Screenshot 2023-04-11 at 10 05 07 AM](https://user-images.githubusercontent.com/22647707/231188618-f37faea4-58ac-4d4b-a60a-0762c9cf9ea9.png)